Makefile

```diff
@@ -143,11 +143,6 @@ $(MOCKGEN_BIN): deps
 	@echo "+ $@"
 	go install github.com/golang/mock/mockgen
 
-GENNY_BIN := $(GOBIN)/genny
```

Comment:
There are still some references in the Makefile for `GENNY_BIN` within `go-generated-srcs` in line 314.

Reply:
thx, fixed